src/frontend/screens/MapScreen/index.tsx

```diff
@@ -112,7 +112,7 @@ export const MapScreen = () => {
         latitude={coords ? coords[1] : undefined}
         bottom={20}
       />
-      {coords !== undefined && isFinishedLoading && (
+      {coords !== undefined && locationServicesEnabled && (
```

Comment:
@achou11 I also realized that if I have the undefined check on user location than I need the undefined check here on the location follow icon as well